### PR TITLE
make badger bench work with txn api in badger

### PR DESCRIPTION
**Pull request by janardhan1993**

### CODE REVIEW

File: bench_test.go [hunk 1]
```diff
@@ -376,7 +377,8 @@ func BenchmarkIterateBadgerOnlyKeys(b *testing.B) {
 			// 100 = size, 0 = num workers, false = fwd direction.
 			opt := badger.IteratorOptions{}
 			opt.PrefetchSize = 10000
```
Comment (manishrjain) [inline bench_test.go]: 256

File: bench_test.go [hunk 2]
```diff
@@ -408,7 +410,8 @@ func BenchmarkIterateBadgerWithValues(b *testing.B) {
...
```
Comment (manishrjain) [inline bench_test.go]: 256

File: bench_test.go (hunk 1 above)
Comment (janardhan1993) [reply bench_test.go]: done

File: bench_test.go (hunk 2 above)
Comment (janardhan1993) [reply bench_test.go]: done

**Pull request merged**